### DISCUSSION TITLE
Spinner

### DISCUSSION THREAD

**Discussion by bamarillo** [saadeghi/daisyui#1320] [Ideas / feature requests]
I'm working on a project that needs a spinner to show that different lists and components are loading and found that there is no spinner, Am I missing something?  I think that a generic spinner that can be placed anywhere you can place a block element would be awesome to have.

**Comment by saadeghi** [comment]
I wanted to add a spinner at some point but then I realized: 
- there are many types of spinners and one design is not enough.
- If the spinner is just a SVG icon and it can be animated using one Tailwind CSS class name, so what's the point of adding it to daisyUI?

You can animate any SVG like this using `animate-spin` class:
```html
<svg
  class="animate-spin"
  xmlns="http://www.w3.org/2000/svg"
	width="40px" height="40px" viewBox="0 0 399.389 399.389" style="enable-background:new 0 0 399.389 399.389;">
	<path d="M340.896,58.489C303.18,20.773,253.031,0.001,199.693,0.001c-53.34,0-103.487,20.771-141.204,58.488
		C20.772,96.207,0,146.355,0,199.694c0,53.34,20.772,103.489,58.49,141.206c37.717,37.717,87.864,58.488,141.204,58.488
		c53.339,0,103.486-20.771,141.205-58.488c37.717-37.717,58.49-87.865,58.49-141.206C399.387,146.355,378.613,96.207,340.896,58.489
		z M328.061,71.326c34.289,34.289,53.172,79.878,53.172,128.368h-41.148c0-77.412-62.979-140.391-140.391-140.391
		c-4.593,0-9.134,0.229-13.615,0.662v-41.31c4.508-0.332,9.049-0.5,13.615-0.5C248.184,18.155,293.771,37.038,328.061,71.326z
		 M199.693,321.931c-67.401,0-122.236-54.835-122.236-122.236S132.292,77.458,199.693,77.458S321.93,132.293,321.93,199.694
		S267.094,321.931,199.693,321.931z"/>
</svg>
```
https://play.tailwindcss.com/ZZmV7uPuj3

Let me know if you have any questions.

**Comment by architjee** [comment]
@bamarillo Hi,
It's true that there's no generic version of a spinner, but we can use radial-progress with animate-spin class. Here's a sample snippet how.
```html
  <div class="flex items-center justify-center">
    <div class="animate-spin radial-progress" style="--value:70;"></div>
  </div>
```
